view/src/app/components/dashboard/service/dashboard.service.ts

```diff
@@ -36,14 +36,19 @@ export class DashboardService {
   saveCurrentDeviceData(uuid: string, timestamp: string, data) {
     const currentDate = this.dateParser(new Date(timestamp));
     console.log(uuid, currentDate, data);
-    return this.http.post('http://localhost:8081/device-data/', {uuid, timestamp: currentDate, data});
+    return this.http.post('http://localhost:8081/device-data/', {uuId: uuid, timestamp: currentDate, data});
```

Comment:
Use property variables instead of sending the request to localhost:8081. This won't work in Heroku.